app/models/card.rb

```diff
@@ -1,2 +1,18 @@
 class Card < ActiveRecord::Base
+  validates :original_text, :translated_text, :review_date, presence: { message: I18n.t('error.validation.messages.cant_be_blank') }
+
+  validate :check_difference
+
+  before_validation :set_review_date
+
+  private
+
+  def check_difference
+    errors.add( I18n.t('error.messages.error') ,I18n.t('error.validation.messages.the_same_value')) if
+        original_text.downcase.present? && translated_text.downcase.present? && original_text.downcase == translated_text.downcase
+  end
+
+  def set_review_date
+    self.review_date = DateTime.now.to_date + 3.days
```

Comment:
Do not use `DateTime.now` without zone. Use one of `Time.zone.now`, `DateTime.now.current`, `DateTime.now.in_time_zone`, `DateTime.now.utc`, `DateTime.now.getlocal`, `DateTime.now.iso8601`, `DateTime.now.jisx0301`, `DateTime.now.rfc3339`, `DateTime.now.to_i`, `DateTime.now.to_f` instead.